Makefile

```diff
@@ -9,7 +9,9 @@ CFLAGS := -O2 -g -Wall -Werror $(CFLAGS)
 REPODIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 UIDGID := $(shell stat -c '%u:%g' ${REPODIR})
 
-CONTAINER_ENGINE ?= docker
+# Prefer podman if installed, otherwise use docker.
+# Note: Setting the var at runtime will always override.
+CONTAINER_ENGINE ?= $(if $(shell command -v podman), podman, docker)
 CONTAINER_RUN_ARGS ?= --user "${UIDGID}"
```

Comment:
This means `podman` will use `--user` though, which is (somehow?) detrimental.

Reply:
correct, `CONTAINER_RUN_ARGS` must be empty when ran with `podman`

Reply:
Noted. Will update!

Reply:
Fixed.

Reply:
works for me